src/Compilers/VisualBasic/BasicAnalyzerDriver/VisualBasicDeclarationComputer.vb

```diff
@@ -151,5 +150,29 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 parameterList.Parameters.Select(Function(p) p.Default),
                 SpecializedCollections.EmptyEnumerable(Of SyntaxNode))
         End Function
+
+        Private Shared Function GetInitializerNodes(propertyStatement As PropertyStatementSyntax) As IEnumerable(Of SyntaxNode)
+            Dim parameterInitializers = GetParameterInitializers(propertyStatement.ParameterList)
+            Dim initializer As SyntaxNode = propertyStatement.Initializer
+            If initializer Is Nothing Then
+                initializer = GetAsNewClauseIntializer(propertyStatement.AsClause)
+            End If
+            Return parameterInitializers.Concat(initializer)
```

Comment:
We filter out the null nodes while creating the declaration info: http://source.roslyn.io/#Microsoft.CodeAnalysis/DeclarationComputer.cs,16

Otherwise, we need too many null checks across the declaration computation.